cross/armcc.txt

```diff
@@ -0,0 +1,11 @@
+[binaries]
+c = 'armcc'
+cpp = 'armcc'
+ar = 'armar'
+strip = 'armar'
+
+[host_machine]
+system = 'WINDOWS'
```

Comment:
These should be in lower case. However this seems to be for a bare metal deployment so it should not say Windows but instead something like `baremetal`.